drivers/adc/ad7124/ad7124.c

```diff
@@ -212,6 +212,27 @@ int32_t ad7124_read_register(struct ad7124_dev *dev,
 	return ret;
 }
 
+/**
+ * @brief Wrap the read register function to give it a modern signature.
+ * @param [in] dev - Driver handler pointer.
+ * @param [in] reg - Address of the register to be read.
+ * @param [out] readval - Pointer to the register value.
+ * @return SUCCESS in case of success, error code otherwise.
+ */
+int32_t ad7124_read_register_wrap(struct ad7124_dev *dev, uint32_t reg,
```

Comment:
ad7124_read_register_wrap ->  ad7124_read_register2 wholdn't sound better?
nd add to ad7124_read_register comment that this function should be used instead.

Reply:
Will do.